src/components/isearch/index.js

```diff
@@ -64,7 +64,7 @@ class ISearch extends Component {
   }
 
   render () {
-    console.log('----RENDERING----');
+    console.log('----RENDERING----', this.props.displayedItems);
```

Comment:
We *must* add a `Technical Task` to the `isearch-ui` backlog (*issues list*) to go through the code and remove all the `console.log` *before* we launch this next week. ***and/or*** create a `debug` variable we can set in the front-end to show the debugging console.logs on-demand.